ReactiveObjC/RACSignal+Operations.h

```diff
@@ -14,7 +14,7 @@
 @class RACMulticastConnection<__covariant ValueType>;
 @class RACScheduler;
 @class RACSequence<__covariant ValueType>;
-@class RACSubject;
+@class RACSubject<__covariant ValueType>;
```

Comment:
Isn't `RACSubject` invariant (neither covariant nor contravariant) as it is both a source and a sink?